### PR TITLE
Update psalm types

### PR DESCRIPTION
**Pull request by Tigrov**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

**Comment by vjik** [comment]
@Tigrov A new psalm error has appeared:

`Error: src/Schema.php:388:71: PossiblyNullArgument: Argument 3 of Yiisoft\Db\Schema\TableSchemaInterface::compositeForeignKey cannot be null, possibly null value provided`

Can you fix it?

File: src/Schema.php [hunk 1]
```diff
@@ -61,17 +42,31 @@
...
+ *     name:string
+ * }
+ * @psalm-type PragmaIndexList = array{
+ *     seq:string,
+ *     name:string,
+ *     unique:string,
+ *     origin:string,
+ *     partial:string
+ * }
+ * @psalm-type PragmaTableInfo = array{
```
Comment (vjik) [inline src/Schema.php:62]:
```suggestion
 * @psalm-type PragmaColumn = array{
```

File: src/Schema.php [hunk 2]
```diff
@@ -33,26 +33,7 @@
...
- *       to:string,
- *       on_update:string,
- *       on_delete:string
- *     }
- *   >
- * >
- * @psalm-type PragmaForeignKeyList = array<
- *   string,
- *   array{
+ * @psalm-type PragmaForeignKeyList = array{
```
Comment (vjik) [inline src/Schema.php:36]:
```suggestion
 * @psalm-type PragmaForeignKey = array{
```

File: src/Schema.php (hunk 1 above)
Comment (Tigrov) [reply src/Schema.php]: Renamed to `ColumnInfo`

File: src/Schema.php (hunk 2 above)
Comment (Tigrov) [reply src/Schema.php]: Renamed to `ForeignKeyInfo`

**Review by vjik — APPROVED** [review]

**Pull request merged**